src/LessMsi.Gui/MainForm.cs

```diff
@@ -192,21 +192,35 @@ public void SetTableViewGridDataSource(IEnumerable<object[]> values)
 			{
 				// If the handle is not created yet we cannot resize columns,
 				// so delay the auto-resizing to when the control is created.
-				msiTableGrid.HandleCreated += MsiTableGrid_HandleCreated;
+				msiTableGrid.HandleCreated += Grid_HandleCreated;
 			}
 		}
 
-		private void MsiTableGrid_HandleCreated(object sender, EventArgs e)
+		private void Grid_HandleCreated(object sender, EventArgs e)

```

Comment:
Rather than add such a specific event and casting here, could we add this to another place where we're already initializing things? Like maybe the end of `MainForm.LoadFile` or something. Isn't that a safe time to do this or am I missing something?

Reply:
Now I see that you're using the Grid_HandleCreated event for two different grids. If we need it for one, I'm fine having it for both. Maybe worth a comment in that handler that it could be either grid though.